spec/integration/lib/auth0/api/v2/api_connections_spec.rb

```diff
@@ -53,16 +53,45 @@
 
   describe '.delete_connection' do
     it { expect { client.delete_connection connection['id'] }.to_not raise_error }
-    it { expect { client.delete_connection '' }.to raise_error(Auth0::MissingConnectionId) }
   end
 
   describe '.update_connection' do
+    let!(:connection_to_update) do
+      client.create_connection(name: "#{SecureRandom.uuid[0..25]}#{entity_suffix}",
+                               strategy: strategy,
+                               options: options,
+                               enabled_clients: enabled_clients)
+    end
     new_name = SecureRandom.uuid[0..25]
     let(:options) { { username: new_name } }
     it do
-      expect(
-        client.update_connection(connection['id'], 'options' => options)['options']
-      ).to include('username' => new_name)
+      expect(client.update_connection(connection_to_update['id'], 'options' => options)['options']).to include(
+        'username' => new_name)
+    end
+  end
+
+  describe '.delete_connection_user' do
+    let(:username) { Faker::Internet.user_name }
+    let(:email) { "#{entity_suffix}#{Faker::Internet.safe_email(username)}" }
+    let(:password) { Faker::Internet.password }
+    let!(:user_to_delete) do
+      client.create_user(username,  'email' => email,
+                                    'password' => password,
+                                    'email_verified' => false,
+                                    'connection' => Auth0::Api::AuthenticationEndpoints::UP_AUTH,
+                                    'app_metadata' => {})
```

Comment:
client.create_connection (strategy: strategy) and client.create_user(email => email) have different formats, is this on purpose?